resources/zib/zib-O2Saturation.xml

```diff
@@ -0,0 +1,159 @@
+      <alias value="O2SaturatieDatumTijd" />
+      <type>
+        <code value="dateTime" />
+      </type>
+      <mapping>
+        <identity value="zib-o2saturation-v3.1-2020EN" />
+        <map value="NL-CM:12.10.3" />
+        <comment value="O2SaturationDateTime" />
+      </mapping>
+    </element>
+    <element id="Observation.value[x]">
+      <path value="Observation.value[x]" />
+      <slicing>
+        <discriminator>
+          <type value="type" />
+          <path value="$this" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="Observation.value[x]:valueQuantity">
+      <path value="Observation.value[x]" />
+      <sliceName value="valueQuantity" />
+      <short value="SpO2Value" />
+      <definition value="The element contains the value of the indirect, peripheral measured O2 saturation. The O2 saturation gives a percentage expressing the extent to which the blood’s hemoglobin is saturated with oxygen, the ratio of oxygen-saturated hemoglobin to hemoglobin. The measurement is done in places where sufficient light can get through the skin, such as a finger, toe or earlobe." />
+      <alias value="SpO2Waarde" />
+      <type>
+        <code value="Quantity" />
+      </type>
+      <mapping>
+        <identity value="zib-o2saturation-v3.1-2020EN" />
+        <map value="NL-CM:12.10.2" />
+        <comment value="SpO2Value" />
+      </mapping>
+    </element>
+    <element id="Observation.value[x]:valueQuantity.system">
+      <path value="Observation.value[x].system" />
+      <fixedUri value="http://unitsofmeasure.org" />
+    </element>
+    <element id="Observation.value[x]:valueQuantity.code">
+      <path value="Observation.value[x].code" />
+      <fixedCode value="%" />
+    </element>
+    <element id="Observation.note.text">
+      <path value="Observation.note.text" />
+      <short value="Comment" />
+      <definition value="Comment on the saturation measurement." />
+      <alias value="Toelichting" />
+      <mapping>
+        <identity value="zib-o2saturation-v3.1-2020EN" />
+        <map value="NL-CM:12.10.4" />
+        <comment value="Comment" />
+      </mapping>
+    </element>
+    <element id="Observation.component">
+      <path value="Observation.component" />
+      <slicing>
+        <discriminator>
+          <type value="value" />
+          <path value="code" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="Observation.component:extraOxygenAdministration">
+      <path value="Observation.component" />
+      <sliceName value="extraOxygenAdministration" />
+    </element>
+    <element id="Observation.component:extraOxygenAdministration.code">
+      <path value="Observation.component.code" />
+      <patternCodeableConcept>
+        <coding>
+          <system value="http://loinc.org" />
+          <code value="74206-4" />
+        </coding>

```

Comment:
I think you reuse this code from the STU3 version. I don't see it in the zib nor FHIR specs:

https://hl7.org/fhir/R4/oxygensat.html

It's "confirmed" that we can use this code in R4?

Reply:
Hmm ... good point. The zib didn't change, so we should be able to re-use this code.

Reply:
Created https://bits.nictiz.nl/browse/ZIB-1476

Reply:
Changed the code according to the zib ticket.